xml/ay_bigfile.xml

```diff
@@ -10274,7 +10274,7 @@ config:type="boolean"&gt;true&lt;/keep_install_network&gt;</screen>
     <example>
      <title>Group Configuration</title>
      <screen>&lt;groups&gt;
-  &lt;group&gt;
+  &lt;group config:type="list"&gt;
```

Comment:
This attribute should be applied to the `<groups>` element:

```xml
<groups config:type="list">
  <group>
    <gid>100<gid>
    <groupname>users</groupname>
    <userlist>bob,alice</userlist>
  </group>
</groups>
```

Reply:
thanks - completely missed that. a bit hard to see with all those ampersands going on :)